infra/scripts/k8s-common-functions.sh

```diff
@@ -65,20 +65,25 @@ function helm_install {
     # has some issues with unbound PVCs (that cause kubectl delete pvc to hang).
     echo "${STEP_BREADCRUMB:-} Helm installing feast"
 
-    if ! time helm install --wait "$RELEASE" "${HELM_CHART_LOCATION:-./infra/charts/feast}" \
-        --timeout 15m \
-        --set "feast-jupyter.image.repository=${DOCKER_REPOSITORY}/feast-jupyter" \
-        --set "feast-jupyter.image.tag=${JUPYTER_GIT_TAG:-$GIT_TAG}" \
-        --set "feast-online-serving.image.repository=${DOCKER_REPOSITORY}/feast-serving" \
-        --set "feast-online-serving.image.tag=${SERVING_GIT_TAG:-$GIT_TAG}" \
-        --set "feast-jobservice.image.repository=${DOCKER_REPOSITORY}/feast-jobservice" \
-        --set "feast-jobservice.image.tag=${JOBSERVICE_GIT_TAG:-$GIT_TAG}" \
-        --set "feast-core.image.repository=${DOCKER_REPOSITORY}/feast-core" \
-        --set "feast-core.image.tag=${CORE_GIT_TAG:-$GIT_TAG}" \
+    helm repo add feast-charts https://feast-charts.storage.googleapis.com
```

Comment:
@pyalex going forward, do you think its better to depend on the helm chart repository on GCS (where we can have specific versions, but can possibly push master), or do you want to use a submodule of the helm chart repo? I am assuming the later, right?

Reply:
I'm totally fine with using public helm chart here (from GCS)